packages/api-generator/src/utils.ts

```diff
@@ -59,7 +59,7 @@ export function addPropData (
     const instancePropObj = componentProps[propName]
 
     ;(propObj as any).default = instancePropObj?.default
-    ;(propObj as any).source = instancePropObj?.source
+    ;(propObj as any).source = instancePropObj?.source ?? kebabName
```

Comment:
This was so props only inherited through types would have no source

Reply:
Ah, good point